src/components/table/table.tsx

```diff
@@ -0,0 +1,45 @@
+import { Component, h, Prop, Element } from '@stencil/core';
+import TabulatorTables from 'tabulator-tables';
+
+export interface Column {
+    title: string;
+    field: string;
+}
+@Component({
+    tag: 'limel-table',
+    styleUrl: 'table.scss',
+    shadow: true,
+})
+export class Table {
+    /**
+     * table data to be displayed
+     */
+    @Prop()
+    public tableData: any[] = [];
+    /**
+     * object for column definitions
+     * {
+     *    title -required
+     *    field- required
+     * }
+     */
+    @Prop()
+    public columns: Column[] = [];
```

Comment:
Same thing here. If the consumer updates the columns, will the table display the new headers? :slightly_smiling_face: